--- /Users/ingo/Silverstripe/ss-3.1/framework/security/Security.php
+++ /Users/ingo/Silverstripe/ss-3.1/framework/security/Security.php
@@ -52,2 +52,3 @@
+	 * @config
 	 */
-	protected static $encryption_algorithm = 'blowfish';
+	private static $encryption_algorithm = 'blowfish';
--- /Users/ingo/Silverstripe/ss-3.1/framework/security/Security.php
+++ /Users/ingo/Silverstripe/ss-3.1/framework/security/Security.php
@@ -52,2 +52,3 @@
+	 * @config
 	 */
-	protected static $encryption_algorithm = 'blowfish';
+	private static $encryption_algorithm = 'blowfish';
--- /Users/ingo/Silverstripe/ss-3.1/framework/security/Security.php
+++ /Users/ingo/Silverstripe/ss-3.1/framework/security/Security.php
@@ -52,2 +52,3 @@
+	 * @config
 	 */
-	protected static $encryption_algorithm = 'blowfish';
+	private static $encryption_algorithm = 'blowfish';
